viewing_party/party.py

```diff
@@ -0,0 +1,162 @@
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        new_movie = {}
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+    else: 
+        new_movie = None
+    return new_movie
+    
+def add_to_watched(user_data, movie):
+    if movie:
+        user_data["watched"].append(movie)
+    else:
+        user_data["watched"]    
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    if movie:
+        user_data["watchlist"].append(movie)
+    else:
+        user_data["watchlist"]
+    return user_data
```

Comment:
Same comment as above.

```suggestion
def add_to_watchlist(user_data, movie):
    if movie:
        user_data["watchlist"].append(movie)

    return user_data
```